docs/HowToGuide/configurations-overview.md

```diff
@@ -59,7 +59,11 @@
 
 - API Operator provides the security to your API. You need to define the security which needs to be applied on the API under "security" extension in the API.
 
-- But if you have not specified the security in the API definition, the API operator will apply the default security for the API.
+- At the time of the installation of the API Operator the default security will be applied. If you need to change the default security configurations either you can edit the security custom resource or you can edit the file within the distribution.
```

Comment:
1. `,` is required after the ...installation of the API Operator 
2. `,` is required after the ...If you need to change the default security configurations

Reply:
Fixed with c46ba03a4a619b503842bcdbed458e8be91c4b03